templates/includes/related-articles-highlight-cards.html

```diff
@@ -11,13 +11,13 @@ <h2 class="highlight-cards__title">{{ title }}</h2>
                         {% image article.teaser_image fill-534x413-c100 format-webp as image_webp %}
                         <source srcset="{{ image_webp.url }}"
                                 type="image/webp"
-                                {% if article.teaser_image.alt_text %}alt="{{ article.teaser_image.alt_text }}"{% endif %}/>
+                                alt="{{ article.teaser_image.alt_text }}"/>
                         {% image article.teaser_image fill-534x413-c100 as base_img %}
                         <img src="{{ base_img.url }}"
                              height="{{ base_img.height }}"
                              width="{{ base_img.width }}"
                              class="highlight-cards__image"
-                             {% if article.teaser_image.alt_text %}alt="{{ article.image.alt_text }}"{% endif %}/>
```

Comment:
So, looking at this further. It won't work as is. I agree with your point about this needing to have an empty alt atleast but `alt_text` does not exist on the `CustomImageMode`, `CustomImageRendition` or anyhwere else in this instance. 

You can grab the image's title like this: https://github.com/wagtail/wagtail/blob/main/wagtail/images/models.py#L816 but that does not really work in this context. But these in this context get their context from the title / description so I think we can just do `alt=""`